### PR TITLE
UIREQ-876: UI tests replacement with RTL/Jest for CancelRequestDialog.js

### PR DESCRIPTION
**Pull request by Dmitriy-Litvinenko**
## Purpose
UI tests replacement with RTL/Jest for CancelRequestDialog.js

## Refs
https://issues.folio.org/browse/UIREQ-876

### CODE REVIEW

**Review by BogdanDenis — APPROVED** [review]

File: src/CancelRequestDialog.test.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    }, {
+      name: 'other cancellation reasons',
+      requiresAdditionalInformation: true,
+    }],
+  };
+  const defaultProps = {
+    request: {
+      id: 'ad038163-7368-44e8-9057-c21bd2b9cd4c',
+      instance: {
+        title: 'title',
```
Comment (Terala-Priyanka) [inline src/CancelRequestDialog.test.js:39]: You can another test that verifies if the modal displays the instance title as expected from props.request.instance.title

File: src/CancelRequestDialog.test.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+  cancelRequestDialogCancel: 'stripes-core.button.confirm',
+  cancelRequestDialogClose: 'stripes-core.button.back',
+};
+const testIds = {
+  cancelRequestDialog: 'cancelRequestDialog',
+  cancelRequestDialogCancel: 'cancelRequestDialogCancel',
+  cancelRequestDialogClose: 'cancelRequestDialogClose',
+};
+
+describe('CancelRequestDialog', () => {
```
Comment (Terala-Priyanka) [inline src/CancelRequestDialog.test.js:23]: add test to assert
1. if Select component is rendered.
2. if 'Confirm' button is disabled after clicking it and the modal is closed after a while?
3. if TextArea component with label 'Additional information for patron' is rendered?

File: src/CancelRequestDialog.test.js [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+  const onCancelRequest = jest.fn();
+  const onClose = jest.fn();
+  const cancellationReasons = {
+    records: [{
+      name: 'cancellation reasons',
+      requiresAdditionalInformation: false,
+    }, {
+      name: 'other cancellation reasons',
+      requiresAdditionalInformation: true,
```
Comment (Terala-Priyanka) [inline src/CancelRequestDialog.test.js:32]: Could you add tests to assert if additional information placeholder is displayed based on "cancellationReasons"?

**Review by Terala-Priyanka — CHANGES_REQUESTED** [review]

File: src/CancelRequestDialog.test.js (hunk 1 above)
Comment (Dmitriy-Litvinenko) [reply src/CancelRequestDialog.test.js:39]: Won't do. Hello we send title to formatted message.

**Review by artem-blazhko — APPROVED** [review]

File: src/CancelRequestDialog.test.js (hunk 1 above)
Comment (Dmitriy-Litvinenko) [reply src/CancelRequestDialog.test.js]: Fixed

File: src/CancelRequestDialog.test.js (hunk 2 above)
Comment (Dmitriy-Litvinenko) [reply src/CancelRequestDialog.test.js:25]: Fixed 1 and 3

**Review by Terala-Priyanka — APPROVED** [review]

**Pull request merged**